### PR TITLE
Chore(build) - remove IE11 support from build

### LINKED ISSUE #2751: Remove support for ie11 from build
Placeholder to remove the [IE11 script](https://github.com/patternfly/patternfly-next/wiki/IE11-Support) from our build

### PR DESCRIPTION
**Pull request by evwilkin**
Closes #2751 - This PR removes ie11 support from the build process.

### CODE REVIEW

**Comment by mcoker** [comment]
@evwilkin looks like you may need to rebase your `v4` branch, there are a lot of updates in the PR that don't pertain to the ie11 scripts. Also we have a blurb about ie11 in the [readme](https://github.com/patternfly/patternfly-next/blob/master/README.md) and an associated wiki page we might want to schedule to update after the major version bump is released.

**Comment by evwilkin** [comment]
>  an associated wiki page we might want to schedule to update after the major version bump is released.

New issue to update Wiki separately: https://github.com/patternfly/patternfly-next/issues/2810

File: src/patternfly/layouts/Flex/examples/Flex.css [hunk 1]
```diff
@@ -1,6 +1,5 @@
-.ws-core-l-flex-border,
-.ws-core-l-flex .pf-l-flex .pf-l-flex,
-.ws-core-l-flex .pf-l-flex__item {
```
Comment (mcoker) [inline src/patternfly/layouts/Flex/examples/Flex.css]: These styles shouldn't be in this PR.

Comment (evwilkin) [reply src/patternfly/layouts/Flex/examples/Flex.css]: Looks like this change got pulled in when rebasing, I've reverted these changes so this file & any changes are no longer part of the PR.

**Pull request merged**